referencia/webpay/README.md

```diff
@@ -1788,3 +1788,100 @@ username  <br> <i> xs:string </i> | Nombre de usuario, del cliente, en el sistem
 Nombre  <br> <i> tipo </i> | Descripción
 ------   | -----------
 result  <br> <i> xs:boolean </i> | Indica si la eliminación se realizó correctamente o no.
+
+## Otros Servicios Webpay OneClick Mall
+
+### Captura Diferida Webpay OneClick Mall
+> Los SDKs no soportan aún los servicios Webpay OneClick Mall.
+
+Una transacción OneClick Mall permite que el tarjetahabiente registre su
+tarjeta de la misma forma en que ocurre con una transacción OneClick, asociando
+dicha inscripción a un comercio padre. Ahora, una vez realizada la inscripción, 
+el comercio padre tiene permitido autorizar transacciones sin captura para
+los comercios “hijo” registrados que tengan habilitado captura diferida.
+Además posterior a la autorización tiene permitido capturar dicho monto reservado.
+La autorización se encarga de validar si es posible realizar el cargo a la
+cuenta asociada a la tarjeta de crédito realizando en el mismo acto la reserva
+de monto de la transacción.
+La captura hace efectiva la reserva hecha previamente o cargo en la cuenta de
+crédito asociada a la tarjeta del titular.
+Estas modalidades, por separado, solo son válidas para tarjetas de crédito.
+
+Para realizar esa captura explícita debe usarse el método `capture()`
+
+#### `capture()`
+
+Este método permite a los comercios OneClick Mall habilitados, poder
+realizar capturas diferidas de una transacción previamente autorizada. El método
+contempla una única captura por cada autorización. Para ello se deberá indicar los
+datos asociados a la transacción de venta y el monto requerido para capturar, el cual
+debe ser menor o igual al monto originalmente autorizado.
+Para capturar una transacción, ésta debe haber sido creada por un código de
+comercio configurado para captura diferida. De esta forma la transacción estará
+autorizada pero requerirá una captura explícita posterior para confirmar la
+transacción.
+
+
+**Parámetros**
+
+Nombre  <br> <i> tipo </i> | Descripción
+------   | -----------
+authorizationCode  <br> <i> xs:string </i> | Código de autorización de la transacción que se requiere capturar. Largo máximo: 6.
+buyOrder  <br> <i> xs:string </i> | Orden de compra de la transacción que se requiere capturar. Largo máximo: 26.
```

Comment:
Is this the child (store) buyOrder or the parent (mall) buyOrder?

Reply:
The documentation I received doesn't specify. 
Webpay Plus Deferred section has an `<aside>` note stating: 
```
El método capture() debe ser invocado siempre indicando el código del comercio
que realizó la transacción. En el caso de comercios Webpay Plus Mall, el código
debe ser el código de la tienda virtual específica.
```
I will check with Dannyela if it works the same way

Reply:
I added a similar notice